### PR TITLE
[Content Enrichment] API added: getAttachment

### PR DESCRIPTION
**Pull request by simranmalik-4**
To be merged with gcs-listmessages

### CODE REVIEW

File: src/main/java/dbaccessor/message/MessageAccessor.java [hunk 1]
```diff
@@ -242,4 +243,24 @@ public boolean checkIfMessageIdBelongsToChatId(long messageId, long chatId) {
...
+     * <li> AttachmentId </li>
+     * <li> FileName </li>
+     * <li> FileType </li>
+     * <li> FileSize </li>
+     * </ol>
+     */
+    public Attachment getAttachmentFromMessageId(long messageId) {
+        String sqlStatment = "SELECT Attachment.AttachmentID as AttachmentID, Attachment.FileName as FileName, Attachment.FileType as FileType, Attachment.FileSize as FileSize FROM Message LEFT OUTER JOIN Attachment ON Message.AttachmentID = Attachment.AttachmentID WHERE Message.MessageID = @messageId";
+        Statement statement = Statement.newBuilder(sqlStatment).bind("messageId").to(messageId).build();
+        ImmutableList<Attachment> resultSet = ImmutableList.<Attachment>builder()
```
Comment (AgamAgarwal) [inline src/main/java/dbaccessor/message/MessageAccessor.java:260]: If you are just going to return the first element, there is no point in copying it to an `ImmutableList`.

File: src/main/java/helper/SuccessResponseGenerator.java [hunk 2]
```diff
@@ -182,4 +182,18 @@
         
         return getResponseBodyForListMessages(listOfMessagesImmutable);
     }
+
+    /**
+     * Renders the given parameters in a Map to return a successful HTTP response for all client requests to the GetAttachment API.
+     */
+    public static Map<String, Object> getSuccessResponseForGetAttachment(Attachment attachment, byte[] blob) {
```
Comment (AgamAgarwal) [inline src/main/java/helper/SuccessResponseGenerator.java:189]: The return type here should be `ImmutableMap`.

File: src/main/java/dbaccessor/message/MessageAccessor.java (hunk 1 above)
Comment (simranmalik-4) [reply src/main/java/dbaccessor/message/MessageAccessor.java]: Yes, sorry :D. Made the change.

File: src/main/java/helper/SuccessResponseGenerator.java (hunk 2 above)
Comment (simranmalik-4) [reply src/main/java/helper/SuccessResponseGenerator.java]: Yes, done.

**Pull request merged**